configuration/redisnative/redis.go

```diff
@@ -0,0 +1,300 @@
+// ------------------------------------------------------------
```

Comment:
Just curious that do we plan to support un-native redis in the feature? If so, `redis/native/xxx` would be better.

Reply:
I change nativeredis to redis